src/project/qml/MuseScore/Project/internal/PropertyItem.qml

```diff
@@ -0,0 +1,110 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+
+RowLayout {
+    id: root
+
+    width: 100
+    spacing: 8
+
+    property int propertyNameWidth: -1
+    property int propertyValueWidth: 100
+    property NavigationPanel navigationPanel: null
+    property var propertyItem
+
+    signal changePositionOfIndex(int index)
+    signal propertyNameChanged(string newValue)
+    signal propertyValueChanged(string newValue)
+
+    StyledTextLabel {
+        Layout.preferredWidth: root.propertyNameWidth
+
+        text: root.propertyItem.propertyName ? root.propertyItem.propertyName : ""
+        font: ui.theme.bodyBoldFont
+        horizontalAlignment: Qt.AlignLeft
+        visible: root.propertyItem.isStandardProperty ? root.propertyItem.isStandardProperty : false
+    }
+
+    TextInputField {
+        Layout.preferredWidth: root.propertyNameWidth
+
+        currentText: root.propertyItem.propertyName ? root.propertyItem.propertyName : ""
+        visible: !root.propertyItem.isStandardProperty
+        hint: qsTrc("project", "Property")
+
+        navigation.name: root.propertyItem.propertyName + "PropertyName"
+        navigation.panel: root.navigationPanel
+        navigation.column: root.propertyItem.index * 3
+        navigation.onActiveChanged: {
+            if (navigation.active) {
+                root.changePositionOfIndex(index)
+            }
+        }
+
+        onCurrentTextEdited: function(newValue) {
+            root.propertyNameChanged(newValue)
+        }
+    }
+
+    TextInputField {
+        Layout.fillWidth: true
+
+        currentText: root.propertyItem.propertyValue ? root.propertyItem.propertyValue : ""
+        hint: root.propertyItem.isStandardProperty ? "" : qsTrc("project", "Value")
+        readOnly: root.propertyItem.isTopPanelProperty
+
+        navigation.name: root.propertyItem.propertyName + "PropertyValue"
+        navigation.panel: root.navigationPanel
+        navigation.column: root.propertyItem.index * 3 + 1
+        accessible.name: root.propertyItem.propertyName + " " + currentText
+        navigation.onActiveChanged: {
+            if (navigation.active && !root.propertyItem.isTopPanelProperty) {
+                root.changePositionOfIndex(index)
+            }
+        }
+
+        onCurrentTextEdited: function(newValue) {
+            root.propertyValueChanged(newValue)
+        }
+    }
+
+    FlatButton {
+        Layout.preferredWidth: root.propertyItem.isTopPanelProperty? 0 : 30
+        Layout.preferredHeight: root.propertyItem.isTopPanelProperty? 0 : 30
```

Comment:
It's better to change the button's visibility rather than its size:

visible: !root.propertyItem.isTopPanelProperty